recipes-connectivity/openssl/openssl_%.bbappend

```diff
@@ -0,0 +1,6 @@
+do_install:append () {
+    sed -i '/\[engine_section\]/d' ${D}${sysconfdir}/ssl/openssl.cnf
+    sed -i '/^pkcs11 =/d' ${D}${sysconfdir}/ssl/openssl.cnf
+    echo "[engine_section]" >> ${D}${sysconfdir}/ssl/openssl.cnf
+    echo "pkcs11 = pkcs11_section" >> ${D}${sysconfdir}/ssl/openssl.cnf
+}
```

Comment:
please add
"dynamic_path = path to libpkcs11.so\n"
"MODULE_PATH = path to softhsm lib\n"
"init = 0\n"